Stripe/STPPaymentContext.m

```diff
@@ -568,7 +568,7 @@ - (void)requestSourceFlowRedirectPayment:(STPSource *)source {
     STPSourceCompletionBlock onRedirectCompletion = ^(STPSource *finishedSource, NSError *error) {
         stpDispatchToMainThreadIfNecessary(^{
             if (error) {
-                [self didFinishWithStatus:STPPaymentStatusError error:error];
+                [self didFinishWithStatus:STPPaymentStatusPending error:nil];
```

Comment:
I was thinking more of errors that occur before polling. Eg. Before we even attempt the redirect we could know there is not network. SFVC also tells you if the initial page load failed? I think it's fine to do pending here though. Maybe we should just think about building some better network offline/retry logic into the pipeline somewhere.